src/Server/Validation/Exception/AlreadyAttached.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace K911\Swoole\Server\Validation\Exception;
+
+class AlreadyAttached extends \RuntimeException
```

Comment:
```suggestion
final class DoubleInitializationException extends \RuntimeException
```

or maybe one of: `ReInitializationException`, `AlreadyInitializedExecption`?